### DISCUSSION TITLE
Can I use multiple attributes for external links?

### DISCUSSION THREAD

**Discussion by JetGum** [datenstrom/community#505] [Ask a question] [answered]
Hi y'all,

hope you're safe and healthy!

Yellow supports Markdown Extra implementation which provides extra attributes for headings, links and other elements.

But **it looks like I can add max. 2 attributes to a link. Other attributes are dropped.**

Example:
`[Datenstrom](https://datenstrom.se){rel=1 rel=2 target=_blank}`

Result:
`<a href="https://datenstrom.se" rel="2" target="_blank">Datenstrom</a>`

Any suggestions how I could add 2x rel (namely, "rel="nofollow" as well as "rel="noopener") to external links?

Cheers,

Carsten

**Comment by schulle4u** [comment, marked as answer]
The valid HTML syntax for multiple values is to separate them by spaces, e. g. 

`<a href="https://datenstrom.se" rel="nofollow noopener">Datenstrom</a>`

Unfortunately this currently is not supported by Markdown Extra. There is an experimental markdown parser which uses the parsedown library, but I'm not sure if it is still being maintained. I suggest to use HTML for the moment.

**Comment by JetGum** [comment]
Thx for your feedback. I will switch my external links to HTML then.

Cheers

Carsten
